lightningd/lightningd.c

```diff
@@ -139,10 +232,16 @@ static bool has_all_daemons(const char* daemon_dir)
 	return !missing_daemon;
 }
 
+/* This routine tries to determine what path the lightningd binary is in.
+ * It's not actually that simple! */
```

Comment:
what does "It's not actually that simple!" mean?

Reply:
Changed to: "It's more complex than you might expect."